docs/insomnia/design-documents.md

```diff
@@ -5,40 +5,71 @@ category: "API Design"
 
-1. Go to the Dashboard
-2. Click "Create" button in upper right hand corner
-3. Select "New Design Document" or a method of importing a
-4. Select how you would like to create a new document.
+If you don't see a Document or Collection you're looking for, ensure you're in the right [Project](/insomnia/projects) by clicking on the Insomnia dropdown menu. Note that the dropdown title will be **Insomnia** if you're in the default Project.  
 
-{:.alert .alert-primary}
-**Note**: When naming your document, the name selected will be displayed on the Dashboard. Spaces and special characters are allowed. 🤗
+![Click Dashboard in the upper left-hand corner to return to a listing of your Documents and Collections.](/assets/images/access-dashboard.png)
+_From inside a Document or Collection, return to your listings by clicking Dashboard._
+
+For information about working with Collections, see [Request Collections](/insomnia/request-collections).
+
+## Create a New Design Document
+
+1. Go to the Dashboard.
+2. Click the **Create** button in upper right hand corner.
+3. From the dropdown menu, select **Design Document** or a method of importing.
+    * If you selected **Design Document**, enter a Document name and click **Create**. Spaces and special characters are allowed in your Document name.
+    * If you selected a method of importing, a modal will open prompting you to import your Document.
 
 ## Rename a Design Document
 
-1. Go to the Dashboard
-2. Open the action menu (visualized by the three dots) on the document you wish to rename.
-3. Select "Rename" action
-4. Rename the document
+Rename a Design Document from either the Dashboard or from within a Design Document.
 
-## Delete a Design Document
+### Rename From the Dashboard
 
-1. Go to the Dashboard
-2. Open the action menu (visualized by the three dots) on the document you wish to rename.
-3. Select the "Delete" action
-4. Follow the steps to delete the document
+1. Open the action menu (visualized by the three dots) on the document you wish to rename.
+2. Select **Rename** action.
+3. Rename the document.
+
+### Rename From Inside a Design Document
+
+1. Click on the dropdown arrow next to the Document name.
+2. Click **Document Settings**.
+3. Rename the document. It will auto-save.
 
 ## Edit a Design Document
 
-1. Go to the Dashboard
-2. Click on the document you wish to edit
-3. You will now be presented with the Editor Mode, and a Preview Pane will display the rendered results of the document you are currently editing.
+Edit the name and description of a Design Document from either the Dashboard or from within a Design Document.
+
+### Edit from the Dashboard
+
+1. Go to the Dashboard.
+2. Click on the Document to get to **Editor Mode**.
+
+### Edit from Inside a Design Document
+
+1. Click on the dropdown arrow next to the Document name.
+2. Click **Document Settings**.
+3. Edit the Document name and description as needed.
```

Comment:
The H2 is more for editing the contents of the design document right? For that it's just the first H3 you've got: click on the document to go to the editor (design tab). This extra section isn't applicable because it talks about renaming and not editing the spec contents, and you've already covered renaming above ("Rename from inside a design doc").

Let me know if that didn't make sense 😂

Reply:
Yeah! Makes sense. Updated.